winml/api/windows.ai.machinelearning.native.idl

```diff
@@ -35,4 +35,11 @@ interface ILearningModelDeviceFactoryNative : IUnknown
     HRESULT CreateFromD3D12CommandQueue(ID3D12CommandQueue * value, [out] IUnknown ** result);
 };
 
+[uuid(c71e953f-37b4-4564-8658-d8396866db0d), object, local]
+interface ILearningModelSessionOptionsNative : IUnknown
+{
+    HRESULT OverrideIntraOpNumThreads(UINT32 intraOpNumThreads);
+    UINT32 GetIntraOpNumThreads();
```

Comment:
This needs to be HRESULT GetIntraOpNumThreads(UINT32* numThreads)

Reply:
Is the getter for testing purposes?

Reply:
Getter is for testing purposes. I will change to your function signature

Reply:
is there any other reason for the getter? In general I'd prefer to not introduce test-only code to our public APIs

Reply:
@martinb35  Rather than having the application using WinML keep track of the number of threads set, the app can interrogate the session options object to get the value. This makes the customer's application source code cleaner.

Reply:
Removed getter from Native IDL and instead added a getter into Native Internal IDL